src/ClusterBootstrap/template/RestfulAPI/config.yaml

```diff
@@ -86,3 +86,4 @@ elasticsearch:
   {% endfor %}
 
 job_mountpoints: {{cnf["job_mountpoints"]}}
+enable_job_resource: {{cnf["enable_job_resource"]}}
```

Comment:
Maybe change name to `enable_job_resource_limit`? The name is not clear to me at first glance.